### PR TITLE
DDF-3626: Add timeout to catalog:removeall command in itests

### PR DESCRIPTION
**Pull request by figliold**
#### What does this PR do?
Increases the timeout for the catalog:removeall commad in itests

#### Who is reviewing it? 
@tbatie 
@emanns95 
@paouelle 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
@clockard
@rzwiefel 
#### How should this be tested? (List steps with links to updated documentation)
CI build
#### What are the relevant tickets?
https://codice.atlassian.net/browse/DDF-3626
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/KarafConsole.java [hunk 1]
```diff
@@ -86,4 +86,8 @@ public String runCommand(String command, Principal... principals) {
   public String runCommand(String command) {
     return executeCommand(command, DEFAULT_ROLES);
   }
+
+  public String runCommand(String command, long timeout) {
```
Comment (paouelle) [inline distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/KarafConsole.java]: ✏️  Javadoc it so we actually know what are the units for the timeout

File: distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java [hunk 2]
```diff
@@ -125,6 +125,8 @@
 
   public static final String REMOVE_ALL = "catalog:removeall -f -p";
 
+  protected static final long REMOVE_ALL_TIMEOUT = TimeUnit.MINUTES.toMillis(5);
```
Comment (paouelle) [inline distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java]: ❓ Should it be public?

Comment (figliold) [reply distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java]: build now

**Comment by figliold** [comment]
build now

**Comment by figliold** [comment]
build now
PR build failed in code not changed in PR.

**Comment by figliold** [comment]
build now

**Pull request merged**